bedrock/base/templates/includes/news-feed.html

```diff
@@ -6,7 +6,7 @@
 {% if LANG.startswith('en') and articles %}
 <section class="news">
   <div class="content">
-    <h2 class="section-title"><span>{{ _('Fresh from our Blog') }}</span></h2>
+    <h2 class="section-title"><span>{{ _('Fresh from our blog') }}</span></h2>
```

Comment:
This string is shared among several pages, but is only shown to English locales.